developers/weaviate/configuration/roles.md

```diff
@@ -136,28 +136,435 @@ This example confers viewer permissions to a user for collections starting with
 
 </Tabs>
 
+import RolePyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.roles.py';
+
 ## Role management
+To manage roles, first we must connect to the client with an API key that has access to do so, such as an admin API key.
```

Comment:
How about:

> To manage roles, the user must have appropriate `role` resource permissions.

I am a bit unsure about this example. 

- It can give the impression that the user has to be admin, or that they need to use an API key (could use username + password)
- It is covered by the AuthN page.